pkg/controllers/config/controller.go

```diff
@@ -45,10 +45,11 @@ func (c *controller) Run(stopCh <-chan struct{}) {
 }
 
 func (c *controller) reconcile(key, namespace, name string) error {
-	logger.Info("reconciling ...", "key", key, "namespace", namespace, "name", name)
 	if namespace != config.KyvernoNamespace() || name != config.KyvernoConfigMapName() {
+		logger.V(4).Info("ignoring ...", "key", key, "namespace", namespace, "name", name)
 		return nil
 	}
+	logger.Info("reconciling ...", "key", key, "namespace", namespace, "name", name)
```

Comment:
```suggestion
        logger := logger.WithValues("key", key, "namespace", namespace, "name", name)
	if namespace != config.KyvernoNamespace() || name != config.KyvernoConfigMapName() {
		logger.V(4).Info("ignoring ...")
		return nil
	}
	logger.Info("reconciling ...")
```

Reply:
Thanks for taking the time to review and suggesting the change @eddycharly , it definitely makes the code clearer. Could you please review it once again? thanks.